analyzer/src/analyze/sql_column.py

```diff
@@ -54,11 +54,12 @@ def dataframe_column_name(self):
         This method helps retrieving the needed columns from the dataframe.
         """
         name = f"{self.table}_{self.column}"
-        if len(name) <= 30:
-            return name
+        hash_ = hashlib.sha1(
+            f"{name}{''.join(str(join) for join in self.joins)}".encode()
+        ).hexdigest()
```

Comment:
I don't have a better solution but it'd be nice to explain why we have to do this to differentiate column names in the df